### PR TITLE
Update docs (episode II)

### PR DESCRIPTION
**Pull request by janden**
Add a GitHub Actions workflow for generating the docs and pushing them to GitHub pages. So far only for `master`.

Depends on #687.

### CODE REVIEW

**Comment by janden** [comment]
So once #687 is merged, we should be able to get this in. Unfortunately, we won't know whether it works 100% until it's in master, but I'm fairly confident it'll do the right thing.

File: .github/workflows/conda.yml [hunk 1]
```diff
@@ -11,12 +11,13 @@ jobs:
         strategy:
             matrix:
                 python-version: [3.6, 3.7]
-                pytorch-version: [1.7.0, 1.8]
+                pytorch-version: [1.7, 1.8]
```
Comment (edouardoyallon) [inline .github/workflows/conda.yml:14]: ahaha thanks

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Each algorithm comes packaged with a frontend and backend. The frontend takes ca
 interfacing with the user. The backend defines functions necessary for
 computation of the scattering transform.
 
-Currently, there are height available frontend–backend pairs, NumPy (CPU), scikit-learn (CPU), pure PyTorch (CPU and GPU), PyTorch<=1.7 (CPU and GPU), PyTorch+scikit-cuda (GPU), PyTorch<=1.7+scikit-cuda (GPU), TensorFlow (CPU and GPU), and Keras (CPU and GPU).
+Currently, there are eight available frontend–backend pairs, NumPy (CPU), scikit-learn (CPU), pure PyTorch (CPU and GPU), PyTorch<=1.7 (CPU and GPU), PyTorch+scikit-cuda (GPU), PyTorch<=1.7+scikit-cuda (GPU), TensorFlow (CPU and GPU), and Keras (CPU and GPU).
```
Comment (edouardoyallon) [inline README.md:36]: woops

File: tests/scattering2d/test_torch_scattering2d.py [hunk 3]
```diff
@@ -28,7 +28,7 @@
 except:
     Warning('torch_skcuda backend not available.')
```
Comment (edouardoyallon) [inline tests/scattering2d/test_torch_scattering2d.py:29]: shouldn't those be done in a separate PR? (i'm ok to keep it this way)

**Review by edouardoyallon — APPROVED** [review]
looks ok (thought i'd have done the test in a separate PR)
